### DISCUSSION TITLE
OSPOlogy Follow-up: The Strategy And Ideas Behind The Spotify FOSS Fund

### DISCUSSION THREAD

**Discussion by anajsana** [todogroup/ospology#116] [OSPO Stories]
Discuss with the community topics related to the upcoming OSPOlogy meeting:

📅 June, 1st ➡️ [The Strategy And Ideas Behind The Spotify FOSS Fund](https://community.linuxfoundation.org/events/details/lfhq-todo-group-presents-the-strategy-and-ideas-behind-the-spotify-foss-fund/)

In this open space, people can:

* Ask questions prior to the meeting starts
* Share notes from the conversation
* *Make proposals for new initiatives that can help the OSPO Ecosystem
* Add useful links and resources related to the topic
* Connect with mentors involved in OSPOs and/or Open Source Management

📝 OSPOlogy Meeting Notes ➡️ [g.doc](https://docs.google.com/presentation/d/1uPqI5LnT7mI5I86ob4u_XBsw7_ztSUaCSWcGVhHTwQI/edit?usp=sharing) & 
[OSPOlogy meeting 10.pdf](https://github.com/todogroup/ospology/files/8815282/OSPOlogy.meeting.10.pdf)

📝 Per Notes  ➡️  [TODO Group - FOSS Found Presentation](https://github.com/todogroup/ospology/files/8815809/TODO.Group.-.FOSS.Found.Presentation.pdf)

**Comment by winterrocks** [comment]
How much vetting Spotify is doing for the projects before the funding is decided?

**Comment by anajsana** [comment]
Question raised via chat:

Curious about the statement that 50% of Spotify contributions to open source were outside of work hours.

Does Spotify have a good way to separate work-related contributions vs. contributions made for personal reasons not tied to work with the same GitHub Username?

**Reply by anajsana** [reply]
Other question: Do you earmark funding to projects or do they decide how to spend the money?

**Comment by anajsana** [comment]
Question raised via chat:

I have a question about the funding part. Paying someone means the receiving end needs a bank account. In some communities, this is a problem because there is no legal entity in the community/collective. Also for tax compliance. Have you considered using something as open-collective or do you vet projects on the fact they need to have an official legal entity receiving money and be tax compliant?

**Comment by winterrocks** [comment]
Could different teams / Business Units get a yearly budget for funding and with some guidelines do their funding? Pros / Cons of that kind of approach?

**Comment by vielmetti** [comment]
I like focusing on "risk reduction" as a specific sustainability goal.

**Comment by anajsana** [comment]
Did you consider sponsoring projects that are already well funded/have corporate backing or focused on smaller projects?

How did you get engineer feedback on projects to sponsor? Voting, poll, etc?

**Comment by anajsana** [comment]
Did you consider transitive dependencies as well or just direct dependencies?

**Comment by winterrocks** [comment]
In general, how open are OSS projects about how much funding they already have?
